docs/guide/basics/examples/absolute-stroke-width-icon/App.js

```diff
@@ -1,14 +1,11 @@
-import { RollerCoaster } from "lucide-react";
+import { RollerCoaster } from 'lucide-react';
 
 function App() {
   return (
     <div className="app">
-      <RollerCoaster
-        size={96}
-        absoluteStrokeWidth={true}
-      />
+      <RollerCoaster size={96} absoluteStrokeWidth={true} />
```

Comment:
I prefer to disable linting rules on all `docs/*/examples/*` files. 
The example code reads a bit easier in the documentation if the props are on a new line. This how it looks now on the site:
<img width="708" alt="image" src="https://github.com/lucide-icons/lucide/assets/11825403/f5cda5d4-1db8-4ecb-ad8f-afd1287bd04d">